### ISSUE TITLE
Tooltip/Popover incorrect placement when on wide scrollable page

### ISSUE THREAD

**Issue by Venex** [twbs/bootstrap#19037]
I created a table with many columns so the page has a large horizontal scroll bar.
Each column does contain a button which triggers a popover on click

The placement of the popover seems to be working just fine when clicking on the first few items of the table. 
Now set the horizontal offset to almost the end of the page and notice popovers are being positioned incorrectly and the arrow is separated from the content (arrow is on correct position, content is not but seems to be located more to the left)

This is my example (now with fixed bootlint errors):
http://plnkr.co/edit/SIO4IMgtdknlMRNt3EQl?p=preview

**Comment by twbs-lmvtfy** [comment]
Hi @Venex!

You appear to have posted a live example (http://run.plnkr.co/plunks/SIO4IMgtdknlMRNt3EQl/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Duplicate of #15590.

**Comment by cvrebert** [comment]
There is still hope in the form of #15686 though.

**Comment by Venex** [comment]
I'm sorry, did not find these issues, thank you for your reply!
